draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -1290,6 +1320,61 @@ The following section details a basic set of test vectors that can be used to co
 
 **NOTE** These fixtures are a work in progress and subject to change
 
+## Mocked random scalars
+
+For the purpose of presenting fixtures for the [ProofGen](#proofgen) operation we present here a way to mock the `calculate_random_scalars` operation ([Random scalars computation](#random-scalars-computation)), used by `ProofGen` to create all the necessary random scalars.
+
+Each proof test vector will define a `SEED` (as a nothing-up-my-sleeve value) and set
+
+```
+mocked_calculate_random_scalars(count) :=
+                             seeded_random_scalars(count, SEED)
+```
+
+in place of `calculate_random_scalars` during the operation's procedure.
+
+**Note** For the [BLS12-381-SHA-256](#bls12-381-sha-256) ciphersuite if more than 170 mocked random scalars are required, the operation will return INVALID. Similarly, for the [BLS12-381-SHAKE-256](#bls12-381-shake-256) ciphersuite, if more than 1365 mocked random scalars are required, the operation will return INVALID. For the purpose of describing [ProofGen](#proofgen) test vectors, those limits are inconsequential.
+
+```
+seeded_scalars = seeded_random_scalars(count, SEED)
```

Comment:
```suggestion
seeded_scalars = seeded_random_scalars(seed, count)
```

Seems a bit more consistent with the rest of the draft?

Reply:
Fixed. Thanks!